pages/blog/index.vue

```diff
@@ -23,29 +23,36 @@ import BlogFeatured from '../../components/blog/BlogFeatured.vue'
 
 // MetaTags
 import getSiteMeta from '../../utils/getSiteMeta'
+import axios from 'axios'
+import { parseMarkdown } from '~/utils/parseMarkdown'
+import readingTime from 'reading-time'
 
 export default {
   components: {
     BlogCard,
     BlogFeatured,
   },
-  async asyncData({ $content }) {
-    const articles = await $content('blog')
-      .where({
-        published: true,
-        featured: false,
-      })
-      .sortBy('createdAt', 'desc')
-      .fetch()
+  async asyncData() {
+    const {
+      data: { data },
+    } = await axios.get(`${process.env.strapiUrl}/api/blogs`, {
+      params: {
+        sort: 'publishedAt:desc',
+        populate: '*',
```

Comment:
What's populate `*`?

Reply:
We need to pass this to populate the deeply nested fields like cover images and meta Images..
These fields are not populated by default.

Reply:
I see, thanks for explaining.